### ISSUE TITLE
Uncaught TypeError: Cannot assign to read only property of object '#<Object>'

### ISSUE THREAD

**Issue by dangh** [davidkpiano/react-redux-form#722]
### The Problem
Cannot change field outside of initial data

### Steps to Reproduce
- Use react-redux-form 1.8.1 (after this commit https://github.com/davidkpiano/react-redux-form/commit/17f06a5e72f275acc5a2c6d8252125c7a6fb32a0)
- Init form with data from redux store using `actions.load()` in `componentWillMount`
- Initial data has field `.city` but not `.cityNotExist`
- Focus/blur `.city` field, work as expected
- Focus/blur `.cityNotExist`, error throwed

### Expected Behavior
- in the form `onSubmit` event, result must have both `city` and `cityNotExist` fields

### Actual Behavior
- only received `city`

### Reproducible Code Example
Please check this repo https://github.com/dangh/test-rrf

<img width="750" alt="screen shot 2017-03-27 at 4 58 36 pm" src="https://cloud.githubusercontent.com/assets/1020347/24348553/ac434976-130e-11e7-95af-2bbfb82ac2dc.png">

**Comment by peterox** [comment]
I am also seeing this problem when trying to 'load' a form

**Comment by peterox** [comment]
I've done some investigating and think I know what is causing this.

**Comment by davidkpiano** [comment]
What have you found?

**Comment by peterox** [comment]
Hi David, I hit save a bit too quick

**Comment by peterox** [comment]
Created a pull request https://github.com/davidkpiano/react-redux-form/pull/727 for this issue.

**Comment by BryceByDesign** [comment]
If it helps. I seem to have no issues on componentWillReceiveProps. However, setting the load action creator on componentWillMount still throws an error.

```
    componentWillMount() {
        const { settings, setDefault } = this.props;
        if (settings) setDefault('settings', settings);
    }
```

Connect

```
export default connect(null, {
    setDefault: (model, value) => actions.load(model, value)
})(FormComponent);
```

There was an error with componentWillReceiveProps as well until I set the proper model path as mentioned [here](https://github.com/davidkpiano/react-redux-form/issues/574#issuecomment-267078465)

> **With nested form state:**
> 
> ```
> const store = createStore(combineReducers({
>   something: somethingReducer,
>   forms: combineForms({ foo: initialFoo })
> });
> ```
> **You now get this shape:**
>
> ```
> {
>   something: ...
>   forms: {
>     forms: { foo: ..., $form: ... }
>   }
> }
> ```

So I updated my Form Component model from **settings** to **forms.settings**

`<Form model="forms.settings"...`

Then no issue with this the componentWillReceiveProps lifecylce method

```
    componentWillReceiveProps(nextProps) {
        const { settings, setDefault } = this.props;
        if (nextProps.settings) {
            setDefault('settings', nextProps.settings);
        }
    }
```

**Comment by davidkpiano** [comment]
I believe this was fixed with #727. Please let me know if the issue persists - I was not able to reproduce it.
